index.js

```diff
@@ -2,6 +2,7 @@
 
 import xtend from 'xtend';
 import bbox from '@turf/bbox';
+import distance from '@turf/distance';
```

Comment:
https://github.com/mapbox/cheap-ruler is faster than turf.distance, but practically for the number of accessible feature's we'd expect to deal with it's probably not significant.

Since we are only ordering by distance and don't need the actual figure, I'm tempted to explore the option of doing simpler screen space distance calculations in pixels and dropping the dependency all together, what do you think? :thinking: :thought_balloon:

Reply:
So screen space distance would be quicker but I think it would give odd results at a lower zoom level. For example say I've got a map of airports close to my location, bit of an odd example but you get the idea.

 As for `cheap-ruler` seems like an over optimisation for the number of features, but happy to switch over if you like. One thing to be aware of is that `@turf/distance` shares some deps with `@turf/bbox` so the bundle size will probably be slightly smaller (haven't actually confirmed this).

Let me know what you think.

Reply:
> I think it would give odd results at a lower zoom level

Hmm, how would the results be odd? In fact using geographic distance might be odd at low zooms since you get stretched circles due to the Mercator projection. Either way I think for the use case it's not a big deal either way.

Reply:
> Let me know what you think.

I agree the performance is probably insignificant at the number of markers shown in this use case.

Reply:
> In fact using geographic distance might be odd at low zooms since you get stretched circles due to the Mercator projection

My thinking was that I'm not that interested in projection if I can't see the map, because that's a graphical concern. Although for users with reduced movement I guess that doesn't apply.

Reply:
Agreed. Whether it's based on screen distance or geographic, I think it will be fine for this feature.